src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -11,6 +11,18 @@
 #include <ucs/sys/compiler.h>
 #include <arpa/inet.h> /* For htonl */
 
+#if HAVE_IBV_EXP_DM
+typedef struct uct_rc_mlx5_tm_dm_pack_data {
+    size_t     length;
+    const void *payload;
+} uct_rc_mlx5_tm_dm_pack_data_t;
+
+typedef struct uct_rc_mlx5_am_dm_pack_data {
+    uct_rc_mlx5_tm_dm_pack_data_t super;
```

Comment:
better to have all struct fields aligned

Reply:
it is aligned by default (no packed attribute is used)

Reply:
I mean fields indentation is different for `uct_rc_mlx5_tm_dm_pack_data_t` and `uct_rc_mlx5_am_dm_pack_data_t`.